src/main/java/com/redhat/devtools/lsp4ij/usages/LSPUsageSearcher.java

```diff
@@ -140,16 +140,13 @@ public void processElementUsages(@NotNull PsiElement element, @NotNull Processor
                 LOGGER.error("Error while collection LSP Usages", e);
             }
 
-            // TODO : as this external reference process can be very slow (for our IJ Quarkus);
-            // the call of processExternalReferences is commented.
-            //  Reactivate it just for some language servers (by using an lsp client featurees)
             // For completeness' sake, also collect external usages to LSP (pseudo-)elements
-            /*LSPExternalReferencesFinder.processExternalReferences(
+            LSPExternalReferencesFinder.processExternalReferences(

```

Comment:
Restored this to an enabled state since the invoked method encapsulates the feature flag check.